.github/workflows/pkgdown.yaml

```diff
@@ -1,51 +1,46 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
```

Comment:
So, in general, this is just overwriting your previous config file with what's currently considered the official GHA pkgdown workflow.